### ISSUE TITLE
Product.agda:124,17: expected sequence of possibly hidden bound identifiers :<ERROR>

### ISSUE THREAD

**Issue by IvanRenison** [agda/agda-stdlib#1961]
I have just installed agda with canal, and installed the stdlib with the [installation guide of this repo](https://github.com/agda/agda-stdlib/blob/master/notes/installation-guide.md)

I a'm traing to compile this hello world:
```agda
module hello-world where

open import IO

main = run (putStrLn "Hello, World!")
```
I get the following error:
```
PS C:\Users\ivan\Desktop\agda_pruebas> agda --compile .\hello-world.agda
Checking hello-world (C:\Users\ivan\Desktop\agda_pruebas\hello-world.agda).
 Checking IO (C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\IO.agda).
  Checking Codata.Musical.Costring (C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\Codata\Musical\Costring.agda).
   Checking Codata.Musical.Colist.Base (C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\Codata\Musical\Colist\Base.agda).
    Checking Codata.Musical.Conat.Base (C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\Codata\Musical\Conat\Base.agda).
     Checking Data.Nat.Base (C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\Data\Nat\Base.agda).
      Checking Relation.Binary.Core (C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\Relation\Binary\Core.agda).
       Checking Data.Product (C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\Data\Product.agda).
C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\Data\Product.agda:124,17-17
C:\Users\ivan\agda_libs\agda-stdlib-1.7.1\src\Data\Product.agda:124,17: expected sequence of possibly hidden bound identifiers
:<ERROR>
```
What can I have wrong with my installation?

Note that I am in windows

**Comment by MatthewDaggitt** [comment]
What version of Agda are you using? If you're using v1.7.1 then you need Agda 2.6.2 I think. If Agda 2.6.3 then v1.7.2 is needed.

**Comment by IvanRenison** [comment]
```
> agda --version
Agda version 2.6.0.1
```

**Comment by IvanRenison** [comment]
I have to install an older version of Agda?

**Comment by jamesmckinna** [comment]
Recommend bumping up to 2.6.2.2 if using 1.7.1: this Agda version fixed some important but subtle bugs over 2.6.2...

**Comment by IvanRenison** [comment]
Thank you for the help
